### PR TITLE
Basic worker image building

### PR DESCRIPTION
**Pull request by cretz**
## What was changed

Added `build-worker-image` command. Intentionally don't have a bunch of `git` commands or support `push`ing at this time, though we can add later if we must (though we may find cloning a repo and pushing a docker image to be better suited for the environment and not the CLI).

### CODE REVIEW

File: cmd/build_worker_image.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+		if len(b.tags) == 0 {
+			return fmt.Errorf("at least one tag required for path version")
+		} else if s, err := os.Stat(b.version); err != nil {
+			return fmt.Errorf("invalid path version: %w", err)
+		} else if !s.IsDir() {
+			return fmt.Errorf("invalid path version: must be dir")
+		} else if !filepath.IsLocal(b.version) {
+			return fmt.Errorf("invalid path version: must be beneath this dir")
+		}
+		// We have to copy the entire
```
Comment (Sushisource) [inline cmd/build_worker_image.go:83]: Copy the entire!

**Review by Sushisource — APPROVED** [review]

File: cmd/build_worker_image.go (hunk 1 above)
Comment (cretz) [reply cmd/build_worker_image.go:83]: Yup, copy the entire everything except the entire comment out of my head

File: cmd/run_scenario.go [hunk 2]
```diff
@@ -47,6 +49,7 @@ func (r *scenarioRunner) addCLIFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&r.runID, "run-id", "", "Run ID for this run")
 	fs.IntVar(&r.iterations, "iterations", 0, "Iterations for the scenario (cannot be provided with duration)")
 	fs.DurationVar(&r.duration, "duration", 0, "Duration for the scenario (cannot be provided with iteration)")
+	fs.DurationVar(&r.waitForServer, "wait-for-server", 0, "Duration to try to connect to server before failing")
```
Comment (bergundy) [inline cmd/run_scenario.go:52]: `connect-timeout` maybe? `wait-for-server` sounds to me like a condition (boolean) not a duration.
I'd accept another name where it's clear that this is a duration.

**Review by bergundy — APPROVED** [review]
Nice, super simple.
I wish we could move some of this into features but I can see that there's a bit less value in this case.

File: cmd/build_worker_image.go (hunk 1 above)
Comment (cretz) [reply cmd/build_worker_image.go:83]: :+1: Will fix in next PR coming shortly after this one.

File: cmd/run_scenario.go (hunk 2 above)
Comment (cretz) [reply cmd/run_scenario.go:52]: :+1: Will fix in next PR coming shortly after this one.

**Pull request merged**